contracts/test-utils/tsconfig.lint.json

```diff
@@ -0,0 +1,7 @@
+{
+    // This file is a workaround that issue: https://github.com/palantir/tslint/issues/4148#issuecomment-419872702
```

Comment:
I don't know how that works. Usually you can't have comments in the JSON file, but that works, so I'll leave it there.

Reply:
@albrow